cmd/guacone/cmd/query_vulnerability.go

```diff
@@ -99,6 +111,7 @@ var queryVulnCmd = &cobra.Command{
 		}
 
 		if opts.vulnerabilityID != "" {
+			tableRows := []table.Row{}
```

Comment:
Can we start moving away from this?
Unnecessary initialization is wasted cycles, allocation, and GC. Nil slices can be range'd, len()'d, printed, and append()'d. This is just allocating a 0 capacity slice that is thrown away with the first append.

If we know the capacity ahead of time, a make() is fine.
If we are marshaling and need to consciously differentiate between null and an empty object, we may need an empty slice.
Otherwise, default to `var X []type` for creating slices.

Reply:
sure I can change that.

Reply:
done